tests/generic/mongocursor-tailable-003.phpt

```diff
@@ -0,0 +1,115 @@
+--TEST--
+MongoCursor::tailable() with getNext() and hasNext() (initial query matches one document)
+--SKIPIF--
+<?php if (getenv('SKIP_SLOW_TESTS')) die('skip slow tests excluded by request'); ?>
+<?php require_once "tests/utils/standalone.inc" ?>
+--FILE--
+<?php
+require_once "tests/utils/server.inc";
+
+function log_getmore($server, $info) {
+    //echo "Issuing getmore\n";
```

Comment:
This is commented out temporarily, as it adds clutter to an already failing test.